[2023-07-17T21:35:42.168+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.data_acquisition manual__2023-07-17T21:35:09.696216+00:00 [queued]>
[2023-07-17T21:35:42.178+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.data_acquisition manual__2023-07-17T21:35:09.696216+00:00 [queued]>
[2023-07-17T21:35:42.178+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-17T21:35:42.193+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): data_acquisition> on 2023-07-17 21:35:09.696216+00:00
[2023-07-17T21:35:42.198+0000] {standard_task_runner.py:57} INFO - Started process 263 to run task
[2023-07-17T21:35:42.201+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'data_acquisition', 'manual__2023-07-17T21:35:09.696216+00:00', '--job-id', '788', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmpzlv00rgm']
[2023-07-17T21:35:42.202+0000] {standard_task_runner.py:85} INFO - Job 788: Subtask data_acquisition
[2023-07-17T21:35:42.243+0000] {task_command.py:410} INFO - Running <TaskInstance: main_dag.data_acquisition manual__2023-07-17T21:35:09.696216+00:00 [running]> on host 7e57cd07412d
[2023-07-17T21:35:42.338+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='data_acquisition' AIRFLOW_CTX_EXECUTION_DATE='2023-07-17T21:35:09.696216+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-17T21:35:09.696216+00:00'
[2023-07-17T21:37:04.325+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/wikipedia/wikipedia.py:389 GuessedAtParserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("lxml"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.

The code that caused this warning is on line 389 of the file /home/***/.local/lib/python3.7/site-packages/wikipedia/wikipedia.py. To get rid of this warning, pass the additional argument 'features="lxml"' to the BeautifulSoup constructor.
[2023-07-17T21:46:44.928+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-07-17T21:46:44.940+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=data_acquisition, execution_date=20230717T213509, start_date=20230717T213542, end_date=20230717T214644
[2023-07-17T21:46:44.990+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-17T21:46:45.023+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
